fw-child/resources/js/child-functions.js

```diff
@@ -245,20 +245,98 @@ const $ = jQuery;
         contained: true,
       });
     }
-
-    if ($('body').hasClass('post-type-archive-variable')) {
-      $(document).on('fw_query_success', function (e, query_item) {
-        let query_items = query_item.find('.fw-query-items');
-
-        if (query_items.data('flex_drawer') == undefined) {
-          query_items.flex_drawer({
+    
+    if ($( '.variable-archive-page' ).length > 0) {
```

Comment:
Maybe add a one line comment above this `if` to easily see to what it applies and what it globally does.